src/Yields.jl

```diff
@@ -29,4 +29,15 @@ include("RateCombination.jl")
 include("NelsonSiegelSvensson.jl")
 
 include("precompiles.jl")
+
+
+function InexactError_hint(io::IO, ex::InexactError)
+    print(io, "\nSuggestion: try calling Periodic($(ex.val), 1) instead.")
```

Comment:
Can we add the rationale? Something like:

> A `Periodic` rate requires also passing a compounding frequency. For examle, call `Periodic($(ex.val), 2)` for a rate compounded twice per period.

Reply:
Sure thing!